lib/utils.pm

```diff
@@ -1279,12 +1279,12 @@ sub shorten_url {
     my $ua = Mojo::UserAgent->new;
 
     my $tx = $ua->post('s.qa.suse.de' => form => {url => $url, wishId => $args{wishid}});
-    if (my $res = $tx->success) {
-        return $res->body;
+    if (my $res = $tx->res->is_success) {
```

Comment:
There is no sense to define `$res` if we don't use it, so we can simplify it to `     if ($tx->res->is_success)`
Or assign `$tx->res` to some variable in order to reuse it.

Reply:
I missed that, changed